hack/e2e-suite/services.sh

```diff
@@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 
 prepare-e2e
 
+echo "Creating test namespace ${NAMESPACE}"
+${KUBECTL} create -f - << __EOF__
+{
+    "kind": "Namespace",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "${NAMESPACE}"
+    },
+    "spec": {
+        "finalizers": [
+            "kubernetes"
```

Comment:
This PR fixes one obvious problem -- the non-existent and unnecessary namespace shouldn't be specified when creating the namespace. That generates an error, if nothing else.